core/opendaq/reader/src/typed_reader.cpp

```diff
@@ -185,7 +193,10 @@ ErrCode TypedReader<ReadType>::readData(void* inputBuffer, SizeT offset, void**
         case SampleType::String:
             return readValues<SampleTypeToType<SampleType::String>::Type>(inputBuffer, offset, outputBuffer, count);
         case SampleType::Invalid:
```

Comment:
Do we want a Struct sample type?

Reply:
I was thinking of that too, because I don't like that the struct descriptor is defined with InvalidSampleType and StructFields not being empty. However, I wanted to avoid updating the OPC UA model. If we can do that, I am willing to update the PR with the struct sample type.